atlasdb-config/src/main/java/com/palantir/atlasdb/factory/Leaders.java

```diff
@@ -111,20 +113,23 @@ public static LocalPaxosServices createLocalServices(
         Map<PingableLeader, HostAndPort> otherLeaders = generatePingables(
                 remotePaxosServerSpec.remoteLeaderUris(), sslSocketFactory, userAgent);
 
-        ExecutorService executor = Executors.newCachedThreadPool(new ThreadFactoryBuilder()
-                .setNameFormat("atlas-leaders-%d")
-                .setDaemon(true)
-                .build());
+        InstrumentedExecutorService instrumentedExecutor = new InstrumentedExecutorService(
```

Comment:
I think we want to specify the name so the metrics aren't `instrumented-delegate-#`